pdns/recursordist/docs/manpages/rec_control.1.rst

```diff
@@ -53,6 +53,9 @@ Options
 
 Commands
 --------
+add-cookies-unsupported *IP* [*IP*...]
```

Comment:
we might need a `remove-cookies-unsupported` command as well. Perhaps in a followup PR.

Reply:
That can be achieved by using clear-cookies for a specific IP. The IP will then be handled as "Unknown" and handled as any other host. I must be missing something.

Reply:
I'll mention the mechanism a bit more clearly in the docs.